### ISSUE TITLE
Setting composer config from chemviewer

### ISSUE THREAD

**Issue by cleblond** [partridgejiang/Kekule.js#88]
Is there anyway to set the composer config when the composer is loaded from the viewer (loadEditor button).  For example I'd like to set "Element specified colors=true" as the default when the composer is opened from a viewer.

**Comment by partridgejiang** [comment]
Yes, you can use the ```editorProperties``` property of ```Viewer``` to set the property values of composer associated with the Viewer before opening it. For example, the following codes create a specified render configs to draw the molecule atoms with their own colors in composer:
```js
var renderConfigs = new Kekule.Render.Render2DConfigs();
renderConfigs.getColorConfigs().setUseAtomSpecifiedColor(true);
chemViewer.setEditorProperties({'renderConfigs': renderConfigs});
```
